lib/pages/home_page.dart

```diff
@@ -62,6 +61,52 @@ class _HomePageState extends State<HomePage> {
           ],
         ),
       ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          // Handle the floating action button press
+        },
+        tooltip: 'Add',
+        child: const Icon(Icons.gamepad_rounded),
+      ),
+      floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
+      bottomNavigationBar: BottomAppBar(
+        shape: CircularNotchedRectangle(),
+        notchMargin: 8.0,
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceAround,
+          children: <Widget>[
+            IconButton(
+              icon: Icon(Icons.home_filled),
+              onPressed: () {
+                // _onItemTapped(0);
+                Navigator.pushNamed(context, '/home');
```

Comment:
There is a problem with this line, because it opens up an infinite number of Home pages, it pushes new home pages to the stack and creates a problem of an oversized stack. is it possible to change this? Or maybe like take out the line if that will work better? Please let me know

Reply:
Possiby pop of the stack, please ask @LeratoKgomo  how she did it.

Reply:
I am going to remove it for now and work on a fix later. But for now we can merge it.

Reply:
in the home page there shouldnt be any routing back to the home page unless under some sort of condition so if you removed it i think that should solve the poblem